services/server/src/common/errors/GenericErrorHandler.ts

```diff
@@ -17,6 +17,8 @@ export default function genericErrorHandler(
       logger.error(`Unexpected server error: ${err.message}`, { error: err });
     }
 
+    logger.debug("❌ Error in genericErrorHandler", { error: err });
```

Comment:
Can you log another message? The current is ambigous because we have the same log in the catch. There it means an error was thrown inside the genericErrorHandler.

Reply:
Yes, it's a last resort. I think at the moment it should not be thrown, but we introduced it because we had some error in the past that was not handled and made the server crash.